lib/api/src/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -2650,7 +2653,7 @@ constructTransaction api knownPools poolStatus apiWalletId body = do
                             makeLeft $
                             toTokenMapAndScript ShelleyKeyS
                                 scriptT
-                                (Map.singleton (Cosigner 0) policyXPub)
+                                (Map.singleton (Cosigner 0) $ fromJust policyXPubM)
```

Comment:
🤔 Having to resort to an approach with `fromJust` seems unfortunate...

Reply:
yeah, fair point. Now no partial function fixing ;-)